### ISSUE TITLE
Cannot make orders in exchange

### ISSUE THREAD

**Issue by clayop** [cryptonomex/graphene-ui#41]
With the newest commit GUI exchange gives the following error when I attempts to make an order

```
Unknown error. Failed to place order for 5 TEST
```

In CLI, it works well

**Comment by svk31** [comment]
This is due to the witness node now enforcing minimum fees for transactions, but the js transaction builder currently sets all fees to 0 unless otherwise specified. @jcalfee you probably need to import the fees schedule and set fees accordingly, or would you like us to specify a fee when adding the transaction?

To bypass this for now you can manually set this in line 47 of transaction_operations.coffee:

``` javascript
unless operation.fee
            operation.fee =
                amount: "100000"
                asset_id: "1.3.0"
```

**Comment by svk31** [comment]
Fixed by 0ce2aa3c230e37ce19fd547348a86bcca1f8591c, you'll still need to use the "nathan" account for now though, find it in Explorer -> Accounts then click on Link.
